science/socnetv/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           qt6 1.0
+
+name                socnetv
+github.setup        socnetv app 3.1 v
+revision            0
+
+categories          science
+platforms           darwin
```

Comment:
Looks good. `platforms darwin` is now implicit in MacPorts, so this line can be dropped.

Reply:
I have now removed this line.